backend/tracing/contextual_tracer.go

```diff
@@ -0,0 +1,36 @@
+package tracing
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+
+	"github.com/grafana/grafana-plugin-sdk-go/internal/tenant"
+)
+
+const (
+	attributeNameTenantID = "tenant_id"
```

Comment:
@wbrowne any opionins on the name? The runner currently uses `stackID` in its own spans:

https://github.com/grafana/grafana-plugin-runner/blob/c6156547208a46136ec4756bbad98926aa19f92a/internal/middleware/tracing.go#L70

However OTEL suggests using snake_case:

https://opentelemetry.io/docs/specs/otel/common/attribute-naming/

Reply:
Yeah fair 👍 We can update that in the runner side, and maybe keep `stackID`/`stackId` for the logs